.gitignore

```
@@ -102,3 +102,6 @@ testbin/*
 kubeconfig*
 dev-tls.crt
 dev-tls.key
+dev-ca.crt
+
+__debug_bin*
```

nit: trailling newline
```suggestion
__debug_bin*

```